serviceinfo-api-server/src/main.rs

```diff
@@ -1,19 +1,22 @@
-use std::{collections::HashSet, str::FromStr};
-
 use anyhow::{Context, Result};
-use serde::{Deserialize, Serialize};
-use tokio::signal::unix::{signal, SignalKind};
-use warp::Filter;
-
 use fdo_data_formats::{
     constants::{FedoraIotServiceInfoModule, HashType, ServiceInfoModule},
     types::{Guid, Hash},
 };
-use fdo_store::Store;
+use fdo_store::{Store, StoreConfig};
 use fdo_util::servers::{
     configuration::serviceinfo_api_server::{ServiceInfoApiServerSettings, ServiceInfoSettings},
-    settings_for, ServiceInfoApiReply, ServiceInfoApiReplyInitialUser, ServiceInfoApiReplyReboot,
+    settings_for, settings_per_device, ServiceInfoApiReply, ServiceInfoApiReplyInitialUser,
+    ServiceInfoApiReplyReboot,
 };
+use serde::{Deserialize, Serialize};
+use std::{collections::HashSet, path::Path, str::FromStr};
+use tokio::signal::unix::{signal, SignalKind};
+use warp::Filter;
+
+#[derive(Debug)]
+struct RtrFailure(anyhow::Error);
```

Comment:
We named this struct like that somewhere else in the code because `RtrFailure` was encapsulating a `Return to Rendezvous` failure. In this case we should name the reject something else.

Reply:
Ahh I thought its returnfailure something like that.. will change it to something better